ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -794,6 +800,10 @@ objects:
           export OPENSHIFT_INSTALL_PRESERVE_BOOTSTRAP=true
         fi
 
+        if has_variant "sno"; then
+          export OPENSHIFT_INSTALL_CLUSTER_PROFILE_OVERRIDE="single-node-production-edge"
```

Comment:
The env var name that was eventually merged is OPENSHIFT_INSTALL_EXPERIMENTAL_CLUSTER_PROFILE